### DISCUSSION TITLE
Is it possible to create a backup of PLC using snap7?

### DISCUSSION THREAD

**Discussion by mateusz-glowinski** [gijzelaerr/python-snap7#404] [Q&A] [answered]
Hi,
I just wonder, if it's possible to create a backup of Siemens PLC using snap7.

**Comment by swamper123** [comment, marked as answer]
Nope, you can't make a backup with snap7.
You could only save DBs and similar, but not Networks and how the logic is connected.

**Reply by mateusz-glowinski** [reply]
Thank you for clear answer!

**Comment by gijzelaerr** [comment]
Closing as answered. No, snap7 cannot create PLC backups. Only data block contents can be saved, not program logic or network configuration.
